docs/model_developers/modules.rst

```diff
@@ -329,3 +329,41 @@ for supplying the agents with the appropriate ``inmanta_plugins`` packages.
 The only exception to this rule is when using the ``inmanta export`` command. It exports a project and all its modules'
 ``inmanta_plugins`` packages to the orchestrator server. When this method is used, the orchestrator does not install any modules
 from the Python package repository but instead contains all Python code as present in the local Python environment.
+
+Configure a project to install modules form a private python package repository
+--------------------------------------------------------------------------------
+
+Modules v2 are Python packages that can be installed from private Python package repositories, which may require authentication. This section explains the steps to install such modules.
+
+Create a file named ``.netrc`` in the orchestrator's file system, specifically in the "inmanta" directory. 
+Add the following content to the file:
+
+.. code-block:: text
+
+  machine <url of the private repository>
+  login token
+  password <the token to access the private repository>
+
+In the ``project.yml`` file of your project, ensure that the ``use_config_file`` option under the ``pip`` section is set to ``True``.
+
+Create a file named ``pip.conf`` and add the following content to it:
+
+.. code-block:: text
+
+  [global]
+  timeout = 60
+  index-url = <url of the repository >
+
+Set the ``PIP_CONFIG_FILE`` environment variable to the path of the ``pip.conf`` file. You can do this by running the following command:
+
+.. code-block:: bash
+
+  $ export PIP_CONFIG_FILE=<path/to/pip.conf>
```

Comment:
I think this procedure should describe how the orchestrator should be configured so that compiles started by the server can access the private python package repository. Setting the environment variable in a bash shell will only work for compiles started using the CLI tool.

Reply:
I don't think I know what actions should be taken on the orchestrator. I expect if we use the global pip config file nothing needs to be done?

Reply:
True. If we use the global config, we don't need to environment variable. Otherwise, we needs to set the environment variable in the `/etc/sysconfig/inmanta-server` file (See: https://docs.inmanta.com/inmanta-service-orchestrator-dev/6/administrators/credentials.html)